op-chain-ops/ether/params.go

```diff
@@ -20,13 +15,8 @@ type Params struct {
 
 var ParamsByChainID = map[int]*Params{
 	1: {
-		// These storage keys were unaccounted for in the genesis state of regenesis 5.
-		map[common.Hash]bool{
-			common.HexToHash("0x8632b3478ce27e6c2251f16f71bf134373ff9d23cff5b8d5f95475fa6e52fe22"): true,
-			common.HexToHash("0x47c25b07402d92e0d7f0cd9e347329fa0d86d16717cf933f836732313929fc1f"): true,
-			common.HexToHash("0x2acc0ec5cc86ffda9ceba005a317bcf0e86863e11be3981e923d5b103990055d"): true,
-		},
-		// Regenesis 4 contained a supply bug.
-		new(big.Int).SetUint64(1637102600003999992),
+		// Regenesis 4 contained a supply bug, and the Saurik bug likely
```

Comment:
Worth expanding on this slightly. Something along the lines of "Regenesis 4 (Nov 11 2021) contained a supply bug such that the total OVM ETH supply was 1.637... ETH greater than the sum balance of every account migrated during the regenesis. A further 0.0012 ETH was incorrectly not removed from the total supply by accidental invocations of the Saurik bug (link)."